.github/workflows/test.yml

```diff
@@ -16,12 +16,10 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - ruby: "2.4"
-          - ruby: "2.5"
-          - ruby: "2.6"
           - ruby: "2.7"
           - ruby: "3.0"
             coverage: "yes"
+          - ruby: "3.1"
```

Comment:
Is there something that prevent us from adding 3.2 while we are here?

Reply:
yes. the used ruby lib to interact with vmware doesn't support ruby 3.2 :(